build.gradle

```diff
@@ -177,7 +178,7 @@ dependencies {
     generatedCompile 'io.springfox:springfox-swagger2:2.7.0'
     generatedCompile 'io.springfox:springfox-swagger-ui:2.7.0'
 
-    developmentOnly("org.springframework.boot:spring-boot-devtools")
+    //developmentOnly("org.springframework.boot:spring-boot-devtools")
```

Comment:
why keep it?

Reply:
We should talk about this as a team and decide if it is useful for us. I'll send an email asking for opinions.

Reply:
We discussed. I am removing it.